src/map_command_handler.hpp

```diff
@@ -190,9 +190,30 @@ class map_command_handler
 		}
 		else if (help_on_unknown_) {
 			utils::string_map symbols;
+		    // Get the input command on a string
+            std::string string_user_ = get_cmd();
```

Comment:
If I recall correctly, Wesnoth's naming convention is to use a final underscore for member variables, _not_ for local variables. Someone else please confirm or correct me though?